src/Autofac/Features/GeneratedFactories/GeneratedFactoryRegistrationSource.cs

```diff
@@ -32,6 +32,9 @@
 
 namespace Autofac.Features.GeneratedFactories
 {
+    /// <summary>
+    /// Registration source for generated factory methods (i.e. when resolving <![CDATA[Func<TService>]]> or some variant).
```

Comment:
```suggestion
    /// Registration source for generated factory methods (i.e. when resolving <see cref="Func{T}"/> or some variant).
```

Reply:
Yeah, I don't why I didn't just do that...